src/components/Home/HomeHeaderBackground.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Box, Container } from '@mui/material';
+
+import SearchForm from './SearchForm';
+import HomeBackground from '../../assets/landscapeImage.jpg';
+
+const HomeHeaderBackground = () => {
+  return (
+    <Box
+      sx={{
+        pt: 16,
+        pb: 12,
+        backgroundImage: `url(${HomeBackground})`,
```

Comment:
dont think the `url(...)` is necessary. just put `HomeBackground`

Reply:
It is required. `{HomeBackground}` wouldn't work.

Reply:
You can do `backgroundImage: HomeBackground,`